setup.py

```diff
@@ -24,7 +24,9 @@
 version = "0.4.0"
 release_status = "Development Status :: 4 - Beta"
 dependencies = [
-    "google-api-core[grpc] >= 1.14.0, < 2.0.0dev",
+    "google-api-core[grpc] >= 1.21.0, < 2.0.0dev",
+    "libcst >= 0.2.5",
+    "proto-plus >= 1.1.0",
     'enum34; python_version < "3.4"',
```

Comment:
Please remove the `enum` dependency (since the library is only compatible with 3.6+)

Reply:
👍 done and noted!